src/common/asb/Asb.c

```diff
@@ -893,18 +893,18 @@ void AsbInitialize(OsConfigLogHandle log)
             SetMaxLogSizeDebugMultiplier(GetMaxLogSizeDebugMultiplierFromJsonConfig(jsonConfiguration, log));
             FREE_MEMORY(jsonConfiguration);
         }
-    }
 
-    RestrictFileAccessToCurrentAccountOnly(g_configurationFile);
+        RestrictFileAccessToCurrentAccountOnly(g_configurationFile);
+    }
 
     OsConfigLogInfo(log, "AsbInitialize: %s", g_asbName);
 
-    if (IsDebugLoggingEnabled())
+    if (IsConsoleLoggingEnabled())
     {
-        OsConfigLogWarning(log, "Debug logging is enabled. To disable debug logging, set 'LoggingLevel' to 6 in '%s'", g_configurationFile);
+        OsConfigLogWarning(log, "AsbInitialize: console logging is enabled. If the log rotator is not enabled this may result in a fill-up of the local storage space");
```

Comment:
nit: Maybe useful to change around the wording here?
```suggestion
        OsConfigLogWarning(log, "AsbInitialize: console logging is enabled. If log rotation is not enabled, this may result in excessive storage use.");
```

Reply:
The actual name is 'rotater' so yes, we may do better. Changing to simple 'rotation'